### PR TITLE
Example in Angular

### PR DESCRIPTION
**Pull request by guzbarraf**
Example of using Final Space API in Angular

### CODE REVIEW

**Comment by guzbarraf** [comment]
**A screenshot of the example code in Angular**

Captura de Pantalla 2020-10-27 a la(s) 22 36 24

**Comment by guzbarraf** [comment]
Done @lelouchB

**Comment by lelouchB** [comment]
> 
> 
> Done @lelouchB

That was superfast @guzbarraf

File: frontend/docs/angular.mdx [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+{/* app.component.html*/}
+<h1>
+  {{title}}
+</h1>
+
+<div class="cont-cards">
+  <div *ngFor="let item of items" class="card">
+    <div class="card shadow--tl">
+      <div class="card__image">
+        <img
```
[the remark this answers is not in the document]
Comment (lelouchB) [reply frontend/docs/angular.mdx:67]: @guzbarraf can you resolve this issue?

**Comment by guzbarraf** [comment]
> That was superfast @guzbarraf

It wasn't quick, I got to work, without realizing that I hadn't asked for the issue

File: frontend/docs/angular.mdx [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+  - api
+  - restapi
+  - rest api
+  - example
+  - angular
+description: A live example of the Final Space API using Angular.
+---
+
+
+<Tabs
```
Comment (lelouchB) [inline frontend/docs/angular.mdx:20]: We don't need this tabs.

Since there is only tab
Can you remove this ?

File: frontend/docs/angular.mdx [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+      for (const key in data){
+        if (data.hasOwnProperty(key)){
+          this.items.push(data[key]);
+        }
+      }
+    });
+  }
+}
+
+{/* app.component.html*/}
```
Comment (lelouchB) [inline frontend/docs/angular.mdx:58]: Can you separate it into two code snippets 
```
{/* app.component.ts*/}
```

```
{/* app.component.html*/}
```

Also for css

**Comment by guzbarraf** [comment]
@lelouchB 
If the PR is approved,
does the project need an example of Angular with Axios?

**Review by lelouchB — COMMENTED** [review]
> 
> 
> @lelouchB
> If the PR is approved,
> does the project need an example of Angular with Axios?

You can add one in the tabs like done with Vue+Axios example, if you want.

File: frontend/docs/angular.mdx (hunk 2 above)
Comment (guzbarraf) [reply frontend/docs/angular.mdx:20]: ok, i remove it

File: frontend/docs/angular.mdx (hunk 3 above)
Comment (guzbarraf) [reply frontend/docs/angular.mdx:58]: ok

**Comment by lelouchB** [comment]
One more thing @guzbarraf 
I was trying to make a project of this example on StackBlitz and CodeSandbox,  but I am a completely stranger to Angular so can you please copy this example to CodeSandbox or StackBltiz so I can fork it and add the link in the example page.


Thanks

**Comment by guzbarraf** [comment]
Hi @lelouchB 

I have updated the PR with the mentioned changes,
removed the tabs,
adding the code snippets
and the example in CodeSandbox for the live example,
but I don't understand the error that Github's Labeler throws,
can you help me with that?

**Review by guzbarraf — COMMENTED** [review]
> One more thing @guzbarraf
> I was trying to make a project of this example on StackBlitz and CodeSandbox, but I am a completely stranger to Angular so can you please copy this example to CodeSandbox or StackBltiz so I can fork it and add the link in the example page.
> 
> Thanks

Done in CodeSandbox in this url [https://codesandbox.io/s/final-space-api-with-angular-1mvg2](url)

**Comment by lelouchB** [comment]
Hey @guzbarraf 
Awesome 

Dont worry about labeler, it was just something I was trying yesterday.
Seems like it is causing unnecessary errors.  I will remove it.

I will review and merge this PR  some time today.

Thanks

**Review by lelouchB — APPROVED** [review]

**Comment by guzbarraf** [comment]
Hi @lelouchB
can you still add hacktoberfest labels to these PRs,
so that they count in the hacktoberfest?

**Pull request closed without merging**